plugins/gateway/apigeex/sharedflowbundle.template/policies/Key-Value-Map-Operations-Get-APIClarity-Tracing.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<KeyValueMapOperations continueOnError="false" enabled="true" name="Key-Value-Map-Operations-Get-APIClarity-Tracing" mapIdentifier="apiclarity-kvm">
+  <DisplayName>Key Value Map Operations Get APIClarity Tracing</DisplayName>
+  <Properties/>
+  <ExclusiveCache>false</ExclusiveCache>
+  <ExpiryTimeInSecs>86400</ExpiryTimeInSecs>
+  <InitialEntries>
+    <Entry>
+      <Key>
+        <Parameter>cacheUpdatedTime</Parameter>
+      </Key>
+      <Value>0</Value>
+    </Entry>
+  </InitialEntries>
+  <Get assignTo="token" index="1">
```

Comment:
I have the impression that we are not using that.

Reply:
I will ask Peshan to review